cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -406,6 +406,7 @@ def test_custom_button_expression(appliance, context, objects, button_group, vis
 
     with appliance.context.use(context):
         view = navigate_to(obj, dest_name)
+        view.wait_displayed()
```

Comment:
Please remove this no need of it. Navigation will take care of this.

Reply:
Unless it will take longer than 10 seconds for the view. I don't think it hurts to have this here.

Reply:
While testing locally I noticed that sometimes the test would fail because the page was not fully displayed, this seemed to help that issue.

Reply:
@jawatts yup `10s` is my point.

@john-dupuy let me help you to understand our `navigation`. @izapolsk and @mshriver already provided us facility to deal with such navigation problems.
https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/appliance/implementations/ui.py#L518
so default `wait_for_view` is `10s`. `view.wait_displayed` default value is `10s` so no need of extra `wait_for_view`. Yup its not hurting.

In your case if you are facing `lag` in view display while navigation. As per me you can try:
```
view = navigate_to(obj, dest_name, wait_for_view=10+)
```

Reply:
@digitronik thanks for the response. I knew about the `wait` arg in `create_view` but was unaware about the `wait_for_view` arg in `navigate_to`. I will update this PR to use this!